pkg/innerring/deploy.go

```diff
@@ -64,12 +64,21 @@ func (x *fsChain) SubscribeToNewBlocks() (<-chan *block.Block, error) {
 		return ch, nil
 	}
 
-	err := x.client.ReceiveBlocks()
+	err := x.client.ReceiveHeaders()
```

Comment:
imo, useless renaming, a neofs dev will always think of it as a source of blocks, while "header" may mean many things

here and all the "block" -> "header" in the PR